### DISCUSSION TITLE
manage multiple big project configurations in netbox

### DISCUSSION THREAD

**Discussion by Mido2026** [netbox-community/netbox#13883] [General]
Hello, 

we start using netbox in our company seriously as SoT for our projects network configurations.
We have various big projects (HW+SW) that are similar (having similar configurations) but not identical.
My  questions:
-	How to manage the list of the projects? Can we use exactly one instance for that and switch to each project via new fields or new objects or other mechanism in netbox? 
-	Else : can we automatically create the desired netbox instances ? 

We need your support.

Thank you and best regards
M.

**Comment by mtinberg** [comment]
Honestly if you think you have a use for multiple instances of Netbox and want guidance, maybe look at commercial offerings for hosting and consultancy to help.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: Mido2026 ***@***.***>
Sent: Monday, September 25, 2023 8:41 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] manage multiple big project configurations in netbox (Discussion #13883)


Hello,

we start using netbox in our company seriously as SoT for our projects network configurations.
We have various big projects (HW+SW) that are similar (having similar configurations) but not identical.
My questions:

  *   How to manage the list of the projects? Can we use exactly one instance for that and switch to each project via new fields or new objects or other mechanism in netbox?
  *   Else : can we automatically create the desired netbox instances ?

We need your support.

Thank you and best regards
M.
